cmake/Buildergocub-software.cmake

```
@@ -12,4 +11,4 @@ ycm_ep_helper(ergocub-software
               TAG master
               COMPONENT core
               FOLDER src
```

```suggestion
              FOLDER src
              DEPENDS YARP
```

See https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-add-a-new-package .